events/serializers.py

```diff
@@ -76,6 +89,7 @@ class Meta:
 
     def create(self, validated_data):
         # event_data = validated_data.pop('event')
+        print(validated_data)
```

Comment:
don't need this ?

Reply:
Sure...will remove...was added for debugging purpose

Reply:
Done